tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -2,6 +2,14 @@
 
 from coalib.bearlib.aspects.collections import aspectlist
 from coalib.bearlib.aspects.Metadata import Metadata
+from coalib.bears.LocalBear import LocalBear
```

Comment:
Actually whatever you define in `conftest.py` is used as settings for other test files in the repository or directory you can say because we have two `conftest files one for coala repo  and one for aspects.